mne/decoding/transformer.py

```diff
@@ -17,7 +18,10 @@
 
 
 class Scaler(TransformerMixin):
-    """Standardize data across channels.
+    u"""Standardize data across channels.
+
+    By default, this makes each time point (within each epoch) have
```

Comment:
`epochs_data : array, shape (n_epochs, n_channels, n_times)`

and the `mean` and `std` are taken along `axis=1`, i.e. the channels